src/test/java/uk/co/ramp/covid/simulation/covid/InfectionLogTest.java

```diff
@@ -44,6 +44,7 @@ public void symptomaticCasesAreLogged() {
             }
         }
 
+        assert infected != null;
```

Comment:
Should this be  assertNotNull (just since we are in the harness we might be better debug output).

Reply:
Agreed. This was an IntelliJ auto-correct which I should have spotted.